frontend/packages/integration-tests-cypress/cy-command-additions/index.ts

```diff
@@ -0,0 +1,3 @@
+import './project';
```

Comment:
Note that we're trying to remove `index.ts` files to get rid of circular dependencies.

Reply:
fixed, renamed to `additions.ts`

Reply:
> fixed, renamed to `additions.ts`

The point is we don't want want files like this at all (regardless of name). You should just import from the module it's declared instead of from an index. Renaming doesn't address the issue.